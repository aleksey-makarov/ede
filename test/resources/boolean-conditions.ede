{ "yes": "graugh", "no": false }
---
{% if true %}
true
{% endif %}

{% if yes %}
{{ yes }}
{% endif %}

{% if no %}
no
{% endif %}

{% if unknown %}
unknown
{% endif %}

{% if no || true %}
{{ no }} || true
{% endif %}

{% if yes == "graugh" %}
yes == "graugh"
{% endif %}

{% if none1 %}
none1
{% elif none2 %}
none2
{% elif none3 %}
none3
{% else %}
none!
{% endif %}

{% if no %}
no
{% elif yes %}
yes!
{% endif %}
