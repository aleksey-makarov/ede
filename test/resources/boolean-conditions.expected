true

True

graugh



False || True
